ape_trezor/_cli.py

```diff
@@ -1,120 +1,135 @@
-import json
+from typing import List
 
-import click  # type: ignore
+import click
 from ape import accounts
-from ape.types import SignableMessage  # type: ignore
-from ape.utils import Abort, notify
+from ape.cli import (
+    ape_cli_context,
+    existing_alias_argument,
+    non_existing_alias_argument,
+    skip_confirmation_option,
+)
 from eth_account import Account
-from trezorlib import ethereum  # type: ignore
-from trezorlib.client import get_default_client  # type: ignore
-from trezorlib.tools import parse_path as parse_hdpath  # type: ignore
+from eth_account.messages import encode_defunct
 
-# NOTE: Must used the instantiated version of `AccountsContainer` in `accounts`
-container = accounts.containers["trezor"]
+from ape_trezor.accounts import TrezorAccount
+from ape_trezor.choices import AddressPromptChoice
+from ape_trezor.client import TrezorClient
+from ape_trezor.hdpath import HDBasePath
 
 
 @click.group(short_help="Manage Trezor accounts")
 def cli():
     """
     Command-line helper for managing Trezor hardware device accounts.
-    You can add accounts using the add method.
+    You can add accounts using the `add` command.
```

Comment:
This part seems superfluous because it only shows up when you do `ape trezor --help` and the `add` command's help message is right underneath it and says the same thing.